target/linux/octeon/patches-5.4/102-ubnt-add-erlite-usg3p-dts.patch

```diff
@@ -0,0 +1,59 @@
+--- /dev/null
++++ b/arch/mips/boot/dts/cavium-octeon/ubnt_e100.dts
+@@ -0,0 +1,15 @@
++/*
++ * Device tree source for EdgeRouter Lite.
++ *
++ * Written by: Aaro Koskinen <aaro.koskinen@iki.fi>
++ *
++ * This program is free software; you can redistribute it and/or modify
++ * it under the terms of the GNU General Public License version 2 as
++ * published by the Free Software Foundation.
++ */
++
++/include/ "ubnt_e100-e120.dtsi"
++
++/ {
++	model = "ubnt,e100";
++};
+--- /dev/null
++++ b/arch/mips/boot/dts/cavium-octeon/ubnt_e120.dts
+@@ -0,0 +1,30 @@
++/*
++ * Device tree source for UniFi Security Gateway 3P
++ *
++ * Written by: Martin Böh <mart.b@outlook.de>
++ *
++ * This program is free software; you can redistribute it and/or modify
++ * it under the terms of the GNU General Public License version 2 as
++ * published by the Free Software Foundation.
++ */
++
++/include/ "ubnt_e100-e120.dtsi"
++#include <dt-bindings/gpio/gpio.h>
++
++/ {
++	model = "ubnt,e120";
++
++	leds {
++		compatible = "gpio-leds";
++
++		white {
++			label = "ubnt:white:dome";
++			gpios = <&gpio 0 GPIO_ACTIVE_HIGH>;
++		};
++
++		blue {
++			label = "ubnt:blue:dome";
++			gpios = <&gpio 1 GPIO_ACTIVE_HIGH>;
++		};
++	};
++};
+--- a/arch/mips/boot/dts/cavium-octeon/Makefile
++++ b/arch/mips/boot/dts/cavium-octeon/Makefile
+@@ -1,5 +1,5 @@
+ # SPDX-License-Identifier: GPL-2.0
+-dtb-$(CONFIG_CAVIUM_OCTEON_SOC)	+= octeon_3xxx.dtb octeon_68xx.dtb
++dtb-$(CONFIG_CAVIUM_OCTEON_SOC)	+= octeon_3xxx.dtb octeon_68xx.dtb ubnt_e100.dtb ubnt_e120.dtb
```

Comment:
So, why wasn't ubnt_e100.dtb included here before?

Reply:
See https://github.com/torvalds/linux/blob/master/arch/mips/boot/dts/cavium-octeon/Makefile
and https://github.com/torvalds/linux/blob/master/arch/mips/boot/dts/cavium-octeon/ubnt_e100.dts

These are additions to an upstream device tree. The only missing component from the existing `ubnt_e100.dts` for the Unifi device are the gpio LEDs, which are not present on the erlite.

I haven't contributed to linux, and I don't know how involved/difficult it is to submit code. Considering the dts requires a small, almost unnecessary change (the USG can still run without the LEDs, although controlling them is a prominent feature of OpenWRT), it may not be worth up-streaming a `ubnt_e120.dts`.